integration/ethereummock/mock_l1_network.go

```diff
@@ -67,24 +61,3 @@ func (n *MockEthNetwork) BroadcastTx(tx obscurocommon.EncodedL1Tx) {
 func (n *MockEthNetwork) delay() time.Duration {
 	return obscurocommon.RndBtwTime(n.avgLatency/10, 2*n.avgLatency)
 }
-
-func printBlock(b *types.Block, m Node) string {
```

Comment:
this means we won't be seeing the entries when miners create blocks? They were super useful for debuging

Reply:
I can add it back 👍